utilities/db-migration/migration_test/verify_test_values/verify_db_values_test.go

```diff
@@ -0,0 +1,186 @@
+package verifytestvalues
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-appstudio/managed-gitops/backend-shared/db"
+	addtestvalues "github.com/redhat-appstudio/managed-gitops/utilities/db-migration/migration_test/add_test_values"
+)
+
+var _ = Describe("Test to verify that the data added to database is still present", func() {
+	Context("Verify Test Values in database", func() {
+		AfterEach(func() {
+			err := db.SetupForTestingDBGinkgo()
+			Expect(err).To(BeNil())
+		})
+		It("Test to verify the data added in database is still present", func() {
+			ctx := context.Background()
+			dbq, err := db.NewUnsafePostgresDBQueries(true, true)
+			Expect(err).To(BeNil())
+			defer dbq.CloseDatabase()
+
+			By("Get cluster user")
+			clusterUser := db.ClusterUser{
+				Clusteruser_id: "test-user-1",
```

Comment:
For the primary keys in this test, I think you can retrieve these values from the constants in `add_db_values.go`, e.g. using `addtestvalues.AddTest_PreClusterUser.Clusteruser_id` here.

Likewise for the other primary keys in this test.